docs/DeveloperGuide.adoc

```diff
@@ -407,7 +444,86 @@ Use case ends.
 Use case resumes at step 2.
 
 [discrete]
-=== Use case: Add new appointment for a patient (UC3)
+=== Use case: Serve next patient (UC3)
+
+*MSS*
+
+1.  Patient exits from room 1
+2.  User requests to allocate patient into room 1
+3.  System removes the patient from queue and allocates him/her to room 1
+
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. Doctor is resting
++
+Use case ends.
+
+* 3a. The given index is invalid.
++
+[none]
+** 3a1. System shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Doctor takes a break (UC4)
+
+*MSS*
+
+1.  Doctor in room 1 takes a break
+2.  User requests to avoid directing patients to the doctor in room 1
+3.  System sets the doctor to be on break
+
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. Doctor is already on break
++
+** 2a1. System shows an error message.
++
+Use case ends.
+
+* 3a. The given index is invalid.
++
+[none]
+** 3a1. System shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Doctor resumes his/her duty (UC5)
+
+*MSS*
+
+1.  Doctor in room 1 resumes his/her duty
```

Comment:
Please note that the MSS should only be concern of the actor "clerk". Please remove this line.

Reply:
Done